index.d.ts

```diff
@@ -0,0 +1,27 @@
+declare module 'webext-options-sync' {
```

Comment:
This is moot and part of the legacy TS module system. Should no longer be used.

Reply:
I haven't done many hand crafted types, usually just generated ones. Do we just completely avoid a module now?

Reply:
Yes